pyproject.toml

```diff
@@ -37,7 +37,6 @@ dependencies = [
   "annotatedyaml==1.0.2",
   "astral==2.2",
   "async-interrupt==1.2.2",
-  "attrs==25.4.0",
```

Comment:
We're still using the old attr namespace, eg here:
https://github.com/home-assistant/core/blob/a2ade413c20a98d52a6f83dc2459ce9d6d66663e/homeassistant/helpers/device_registry.py#L14

Reply:
Should those imports be replaced with `attrs`?